Project.swift

```diff
@@ -61,7 +61,17 @@ let project = Project(name: "Mail",
                                   infoPlist: "MailTests/Info.plist",
                                   sources: "MailTests/**",
                                   dependencies: [
-                                      .target(name: "Infomaniak Mail")
+                                      .target(name: "Infomaniak_Mail")
+                                  ],
+                                  settings: .settings(base: Constants.testSettings)),
+                          .target(name: "MailAPITests",
```

Comment:
`MailAPITests` are within a separate target, as I want to be able to test the app in isolation of the API and network, like done in Drive.